src/sendgrid.lisp

```diff
@@ -124,6 +140,27 @@ The JSON looks like:
   (assert (string-equal (sendgrid-json :to '("to@mail" "to-two@mail") :from "me@mail" :subject "hello" :content-value "yo" :reply-to '(("email" . "@") ("name" . "me")))
                         "{\"personalizations\":[{\"to\":[{\"email\":\"to@mail\"}],\"to\":[{\"email\":\"to-two@mail\"}]}],\"from\":{\"email\":\"me@mail\"},\"reply_to\":{\"email\":\"@\",\"name\":\"me\"},\"subject\":\"hello\",\"content\":[{\"type\":\"text/plain\",\"value\":\"yo\"}]}")))
 
+
+;; Logic For Sending Attachments
+
+(defun file-to-seq (file)
+  "convert a file into a sequence of bytes"
+  (flexi-streams:string-to-octets
+   (alexandria:read-file-into-string file)))
```

Comment:
I don't think you need alexandria for this. See `uiop:read-file-string`.

Reply:
This is more habitual. I tend to use Alexandria because I have reviewed the code and trust it. I just read uiop:read-file-string and I agree.